src/components/CurrentWallet.jsx

```diff
@@ -77,17 +83,18 @@ export default function CurrentWallet ({ currentWallet }) {
         </div>}
       {walletInfo && walletInfo?.total_balance &&
         <>
-          <p>Total Balance: {valueToUnit(walletInfo.total_balance, unit)}</p>
+          <p>Total Balance: {valueToUnit(walletInfo.total_balance, unit, redactBalances)}</p>
+          <rb.Form.Check type="switch" label="Show Balances" checked={!redactBalances} onChange={(e) => setAndPersistRedactBalances(!e.target.checked)} />
           <rb.Form.Check type="switch" label="Display amounts in SATS" checked={unit === SATS} onChange={(e) => setAndPersistUnit(e.target.checked ? SATS : BTC)} />
```

Comment:
Let's apply a margin at the botton here:

```suggestion
          <rb.Form.Check type="switch" label="Display amounts in SATS" checked={unit === SATS} onChange={(e) => setAndPersistUnit(e.target.checked ? SATS : BTC)} className="mb-4" />
```